package.json

```
@@ -1,7 +1,7 @@
 {
   "name": "@nordcloud/gnui",
   "description": "Nordcloud Design System - a collection of reusable React components used in Nordcloud's SaaS products",
-  "version": "6.0.0",
+  "version": "6.1.0",
```

Major version needs to be increased since we're introducing a breaking change
https://semver.org/#spec-item-8

```suggestion
  "version": "7.0.0",
```

---

Thx, fixed